### PR TITLE
Add basic support for HitachiAirToAirHeatPump

### PR DESCRIPTION
**Pull request by iMicknl**
Based on https://github.com/dotvav/aasivak/blob/master/Aasivak.py and https://pastebin.com/HZKsEPjU. 

Great work by @aasivak, which I try to port bit by bit... Feel free to chime in.

### CODE REVIEW

**Comment by xorassistant** [comment]
Hi @iMicknl

We also have an Hitachi Heating System(**HitachiAirToWaterMainComponent**), this is similair to the one you have put into the Tahoma integration.

Only the device is not supported the following DEBUG is coming forward:
[DEBUG-HitachiAirToWaterMainComponent.txt](https://github.com/iMicknl/ha-tahoma/files/5964991/DEBUG-HitachiAirToWaterMainComponent.txt)


`2021-02-11 12:45:45 DEBUG (MainThread) [custom_components.tahoma.coordinator] Initialized DataUpdateCoordinator with 0:00:30 interval.
2021-02-11 12:45:45 DEBUG (MainThread) [custom_components.tahoma.coordinator] EventName.DEVICE_STATE_CHANGED/None (device: modbus://1233-4770-7251/4002825/1#1, state: None -> None)
2021-02-11 12:45:45 DEBUG (MainThread) [custom_components.tahoma] Finished fetching TaHoma Event Fetcher data in 0.052 seconds
2021-02-11 12:45:45 INFO (MainThread) [custom_components.tahoma] HomeKit support detected with setup code 145-51-529.
2021-02-11 12:45:45 DEBUG (MainThread) [custom_components.tahoma] Added device (internal:TSKAlarmComponent - Alarm - TSKAlarmController - internal://1233-4770-7251/alarm/0)
2021-02-11 12:45:45 DEBUG (MainThread) [custom_components.tahoma] Unsupported device detected (modbus:YutakiMainComponent - HitachiHeatingSystem - HitachiAirToWaterMainComponent - modbus://1233-4770-7251/4002825/1#1)
2021-02-11 12:45:45 DEBUG (MainThread) [custom_components.tahoma] Unsupported device detected (modbus:YutakiV2Zone1Component - HitachiHeatingSystem - HitachiAirToWaterHeatingZone - modbus://1233-4770-7251/4002825/1#2)
2021-02-11 12:45:45 DEBUG (MainThread) [custom_components.tahoma] Unsupported device detected (modbus:YutakiRoomThermostatZone1Component - HeatingSystem - HitachiThermostat - modbus://1233-4770-7251/4002825/1#6)
2021-02-11 12:45:45 DEBUG (MainThread) [custom_components.tahoma] Unsupported device detected (modbus:YutakiRoomThermostatZone2Component - HeatingSystem - HitachiThermostat - modbus://1233-4770-7251/4002825/1#7)
2021-02-11 12:45:45 DEBUG (MainThread) [custom_components.tahoma] Added device (modbus:YutakiV2SpaceHeatingElectricalEnergyConsumptionComponent - ElectricitySensor - CumulativeElectricPowerConsumptionSensor - modbus://1233-4770-7251/4002825/1#9)
2021-02-11 12:45:45 DEBUG (MainThread) [custom_components.tahoma] Added device (ovp:ModbusMainController - Generic - unknown - ovp://1233-4770-7251/4002825)
2021-02-11 12:45:45 DEBUG (MainThread) [custom_components.tahoma] Added device (rts:VenetianBlindRTSComponent - VenetianBlind - UpDownVenetianBlind - rts://1233-4770-7251/16755180)
2021-02-11 12:45:45 DEBUG (MainThread) [custom_components.tahoma] Added gateway (1233-4770-7251 - GatewayType.TAHOMA_V2 - GatewaySubType.TAHOMA_PRO)
2021-02-11 12:46:18 DEBUG (MainThread) [custom_components.tahoma.coordinator] EventName.DEVICE_STATE_CHANGED/None (device: modbus://1233-4770-7251/4002825/1#1, state: None -> None)
2021-02-11 12:46:18 DEBUG (MainThread) [custom_components.tahoma.coordinator] EventName.DEVICE_STATE_CHANGED/None (device: modbus://1233-4770-7251/4002825/1#1, state: None -> None)
2021-02-11 12:46:18 DEBUG (MainThread) [custom_components.tahoma] Finished fetching TaHoma Event Fetcher data in 0.121 seconds`

Do you see a way to also put this Hitachi into the intergration?

File: custom_components/tahoma/tahoma_entity.py [hunk 1]
```diff
@@ -111,7 +111,10 @@ def device_info(self) -> Dict[str, Any]:
             }
 
         manufacturer = self.select_state(CORE_MANUFACTURER_NAME_STATE) or "Somfy"
-        model = self.select_state(CORE_MODEL_STATE) or self.device.widget
+        model = (
+            self.select_state(CORE_MODEL_STATE, "core:ProductModelNameState")
```
Comment (tetienne) [inline custom_components/tahoma/tahoma_entity.py:115]: Missing the constant. This state  come from Hitachi devices?

**Review by vlebourl — APPROVED** [review]

File: custom_components/tahoma/tahoma_entity.py [hunk 2]
```diff
@@ -160,6 +164,20 @@ def select_attribute(self, *attributes) -> Optional[str]:
                 ),
                 None,
             )
+
+    def select_state_definition(self, *states) -> Optional[str]:
```
[the remark this answers is not in the document]
Comment (iMicknl) [reply custom_components/tahoma/tahoma_entity.py:168]: Indeed, didn't mark it as todo, but will remove it if I won't end up using it. I was experimenting with the state definitions, to see if I could pull the fan modes from there directly.

But as usual with Somfy, they show 'auto cooling', but return 'autoCooling' via tahomalink.com...

File: custom_components/tahoma/tahoma_entity.py (hunk 1 above)
Comment (iMicknl) [reply custom_components/tahoma/tahoma_entity.py:115]: If I end up using this, I will make it a constant. However, I am still a bit in doubt here. It is a constant used by Hitachi devices, however it is changeable by the user...

**Pull request merged**